qa/L0_backend_python/test.sh

```diff
@@ -298,44 +308,53 @@ done
 kill $SERVER_PID
 wait $SERVER_PID
 
-(cd env && bash -ex test.sh)
-if [ $? -ne 0 ]; then
-  RET=1
+# Disable env test for Jetson since non-dockerized testing might cause issues during failure.
+if [ "$TEST_JETSON" == "0" ]; then
+  (cd env && bash -ex test.sh)
+  if [ $? -ne 0 ]; then
+    RET=1
+  fi
 fi
 
 (cd lifecycle && bash -ex test.sh)
 if [ $? -ne 0 ]; then
   RET=1
 fi
 
-(cd ensemble && bash -ex test.sh)
-if [ $? -ne 0 ]; then
-  RET=1
-fi
+# Disable ensemble, unittest, io and bls tests for Jetson since GPU Tensors are not supported
+if [ "$TEST_JETSON" == "0" ]; then
+  (cd ensemble && bash -ex test.sh)
+  if [ $? -ne 0 ]; then
+    RET=1
+  fi
 
-(cd restart && bash -ex test.sh)
-if [ $? -ne 0 ]; then
-  RET=1
-fi
+  (cd unittest && bash -ex test.sh)
+  if [ $? -ne 0 ]; then
+    RET=1
+  fi
 
-(cd unittest && bash -ex test.sh)
-if [ $? -ne 0 ]; then
-  RET=1
-fi
+  (cd io && bash -ex test.sh)
+  if [ $? -ne 0 ]; then
+    RET=1
+  fi
 
-(cd io && bash -ex test.sh)
-if [ $? -ne 0 ]; then
-  RET=1
+  (cd bls && bash -ex test.sh)
+  if [ $? -ne 0 ]; then
+    RET=1
+  fi
 fi
 
-(cd variants && bash -ex test.sh)
+(cd restart && bash -ex test.sh)
 if [ $? -ne 0 ]; then
   RET=1
 fi
 
-(cd bls && bash -ex test.sh)
-if [ $? -ne 0 ]; then
-  RET=1
+# Skip variants for Jetson since already built without GPU Tensors support
+if [ "$TEST_JETSON" == "0" ]; then
+  (cd variants && bash -ex test.sh)
```

Comment:
Why are there separate blocks for skipping on jetson? Why not have a single $TEST_JSON == "0" block and collect all the tests to be skipped there?

Reply:
Done